onnx/utils.py

```diff
@@ -79,6 +79,14 @@ def _collect_reachable_nodes(
         nodes = [n for n in self.graph.node if n in reachable_nodes]
         return nodes
 
+    def _collect_referred_local_functions(
+            self,
+            nodes,  # type: List[NodeProto]
+    ):  # type: (...) -> List[FunctionProto]
+        node_op_types = {n.op_type for n in nodes}
+        local_functions = [f for f in self.model.functions if f.name in node_op_types]
```

Comment:
I think we may need to check both (domain, op_type) to match calling-node with called-function?